parcel.go

```diff
@@ -14,47 +14,108 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
 	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
+	queryText := "INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)"
 
+	res, err := s.db.Exec(queryText, sql.Named("client", p.Client),
+		sql.Named("status", p.Status), sql.Named("address", p.Address), sql.Named("created_at", p.CreatedAt))
+
+	if err != nil {
+		return -1, err
+	}
+
+	lastInd, err := res.LastInsertId()
+	if err != nil {
+		return -1, err
+	}
 	// верните идентификатор последней добавленной записи
-	return 0, nil
+	return int(lastInd), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
 	// реализуйте чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
+	queryText := "SELECT client, status, address, created_at, number FROM parcel WHERE number = :parcelNumber"
+
+	res := s.db.QueryRow(queryText, sql.Named("parcelNumber", number))
 
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := res.Scan(&p.Client, &p.Status, &p.Address, &p.CreatedAt, &p.Number)
+	if err != nil {
+		return p, err
```

Comment:
Если ошибка, то лучше возвращать пустое значение
```suggestion
		return Parcel{}, err
```